constants/constants.go

```diff
@@ -2,6 +2,6 @@ package constants
 
 const (
 	Protocol          = "dgit"
```

Comment:
Thoughts on changing the protocol to `dg://` also?

Reply:
Sounds good to me. Do we need migrations for that? Or are we already in the land of repo recreation here?

Reply:
i think its part of repo recreation of this release -- essentially the old `dgit` remote is no longer used with the following two lines, so its so fresh and so clean for `dg`